<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle De Nível</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Conexão</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="HostLabel">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Host</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="HostInput">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>100</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>192.168.43.44</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10.0.0.101</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="PortaLabel">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Porta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PortaInput">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>4000</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="Desconectar">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Desconectar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="Conectar">
          <property name="text">
           <string>Conectar</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>WRITE</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Controle</string>
       </attribute>
       <widget class="QWidget" name="sinalControle" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>531</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="SinalSaida" native="true">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>0</y>
          <width>541</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>310</y>
          <width>781</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>KP</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="Kp"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>KI</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="Ki"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>KD</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="Kd"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="ref"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Referencia</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="3">
          <widget class="QPushButton" name="controle">
           <property name="text">
            <string>Iniciar Controle</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Identificação</string>
       </attribute>
       <widget class="QWidget" name="sinalControle_2" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>551</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="SinalSaida_2" native="true">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>0</y>
          <width>521</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>320</y>
          <width>771</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Nível Minimo</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="NivelMin">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="valorPWM">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Ajuste do Sinal de PWM =</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="NivelMax">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Nivel Máximo</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="4">
          <widget class="QScrollBar" name="horizontalScrollBar">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="2" colspan="2">
          <widget class="QPushButton" name="identificacao">
           <property name="text">
            <string>Iniciar A Aquisição de Informações</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
